x-pack/plugins/cloud_security_posture/server/routes/configuration/update_rules_configuration.ts

```diff
@@ -106,6 +106,25 @@ export const updatePackagePolicy = (
   return packagePolicyService.update(soClient, esClient, packagePolicy.id, updatedPackagePolicy);
 };
 
+export const setRulesOnPackage = async (
```

Comment:
nit: the name here is not fully describing what actually being done. 
`Set` usually being used for setting a property. usually in-memory
`Update` - has more context that it is going to be taken affect in a persistent storage

In this case I would look for a name that describing an update is going to be made

Now, `rules on package` isn't describing what is being done here.
A more suited name IMO is `updateAgentConfiguration`. Naming the function by its GOAL gives the sense it is not related to implementation. We do want to update the agent's configuration when we disable/ enable a rule.

And reading the implementation, describes HOW it is being done.